helm-charts/templates/configuration.yaml

```diff
@@ -8,25 +8,29 @@ data:
   config.json: |-
     {
       "phovea_server": {
-        "debug": false,
-        "port": 9000,
+        "debug": {{ .Values.ordino.server.debug }},
+        "port": {{ .Values.ordino.server.port }},
       },
       "tdp_core": {
-        "host": "localhost"
+        "host": "mongodb://{{ .Values.ordino.mongodb.username }}:{{ .Values.ordino.mongodb.password }}@{{ .Values.ordino.mongodb.host }}/targid{{ .Values.ordino.mongodb.extraArgs }}",
+        "port": {{ .Values.ordino.mongodb.port }},
+        "database": "targid"
       },
       "phovea_data_mongo": {
-        "host": "localhost"
+        "host": "mongodb://{{ .Values.ordino.mongodb.username }}:{{ .Values.ordino.mongodb.password }}@{{ .Values.ordino.mongodb.host }}/graph{{ .Values.ordino.mongodb.extraArgs }}",
+        "port": {{ .Values.ordino.mongodb.port }},
+        "db": "graph"
       },
       "phovea_data_redis": {
         "assigner": {
-          "hostname": "localhost"
+          "hostname": "{{ .Values.ordino.redis.host }}"
         },
         "mapping": {
-          "hostname": "localhost"
+          "hostname": "{{ .Values.ordino.redis.host }}"
         }
       },
       "tdp_publicdb": {
         "dburl": "postgresql://USER:PASS@HOST/DBNAME",
```

Comment:
How would you use the postgres credentials from _postgrescredentials.yaml_ here? Can you please modify this DB URL string?

Reply:
The postgres credentials cannot be used for now. For this to work I will need an image that will use a Postgres db. After that I can create init scripts for the db and the needed ordino server postgres configuration.

Reply:
We had good success by using conditional includes. That allows us to have, e.g., external endpoints where the persistence lives outside of Kubernetes and still enables you to use "Kubernetes internal" deployments/statefulsets or whatever kind of resource you're using.